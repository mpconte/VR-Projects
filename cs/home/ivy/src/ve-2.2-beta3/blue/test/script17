#
# Test logical operators
#
[stdout] writeln "OR test"
if { 0 || 1 } {
	[stdout] writeln "pass1"
} else {
	[stdout] writeln "fail1"
}
if { 1 || 0 } {
	[stdout] writeln "pass2"
} else {
	[stdout] writeln "fail2"
}
if { 1 || 1 } {
	[stdout] writeln "pass3"
} else {
	[stdout] writeln "fail3"
}
if { 0 || 0 } {
	[stdout] writeln "fail4"
} else {
	[stdout] writeln "pass4"
}

[stdout] writeln "AND test"
if { 0 && 1 } {
	[stdout] writeln "fail1"
} else {
	[stdout] writeln "pass1"
}
if { 1 && 0 } {
	[stdout] writeln "fail2"
} else {
	[stdout] writeln "pass2"
}
if { 1 && 1 } {
	[stdout] writeln "pass3"
} else {
	[stdout] writeln "fail3"
}
if { 0 && 0 } {
	[stdout] writeln "fail4"
} else {
	[stdout] writeln "pass4"
}
