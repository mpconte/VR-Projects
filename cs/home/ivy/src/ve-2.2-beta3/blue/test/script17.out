OR test
pass1
pass2
pass3
pass4
AND test
pass1
pass2
pass3
pass4
result = ok
